src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -1,20 +1,11 @@
 echo off
 setlocal
 
-echo BridgeKeepRunning=%BridgeKeepRunning%
-if '%BridgeKeepRunning%' neq 'true' (
-    echo Releasing Bridge resources and stopping it if running locally.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
-    call Bridge.exe -stopIfLocal -reset %*
-    popd
-) else (
-    echo Releasing Bridge resources but leaving it running.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -reset %* ...
-    call Bridge.exe -reset %*
-    popd
-    echo The Bridge was left running because BridgeKeepRunning is true
-)
+echo Releasing Bridge resources and stopping it if running locally.
+pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+echo Invoking Bridge.exe -stopIfAutoStart -reset %* ...
+call Bridge.exe -stopIfAutoStart -reset %*
```

Comment:
Just a thought: I think our use of binary "true/false" operation modes might be hindering our thinking. 

For example, `stopIfAutoStart` is kind of weirdly worded to me, and I wonder if we should do something like "script mode" vs. "manual mode" vs. "long-running mode"

i.e., 
if you're running under "script" mode, then you expect a script to kill the Bridge (or it to kill itself after a while)
if you're running "manually" you are expected to start/stop yourself
if you're running under "longrunning timeout" mode - it can time out by itself after inactivity

And you'd specify this by using an option like `-mode=manual` or `mode=longrunning`

Not blocking for checkin